plaso/cli/helpers/artifact_definitions.py

```diff
@@ -86,11 +95,19 @@ def ParseOptions(cls, options, configuration_object):
       raise errors.BadConfigOption(
           'Unable to determine path to artifact definitions.')
 
+    custom_artifacts_path = getattr(
```

Comment:
github does not allow me to create comment on 122 but you'll need to set custom_artifacts_path in the configuration object

Reply:
I don't think I need to as I'm processing the file into the artifacts_registry within this helper?

Reply:
Ack, I see you just load the artifact definitions from the path